fractal-puppy/Engine
# Improved usability

Added some new buttons helpful for making objects static and navigable, fixed some bugs of the static objects and added some brofiler flags in order to help with profiling.
I would've splitted all these into different pull requests, but every PR is really hard to get accepted.
To test:
- load the graveyard scene
- select an object with childs that have mesh and  see the new buttons:
        - "make childs with mesh static"
        - "make childs with mesh navigable"
        - "make childs with mesh obstacles"
        - "add all navigable childs to the navmesh"
- select the game object "RecastMesh"
- click on "Make childs with mesh navigable"
- click on "Add all navigable childs to the navMesh"
- click on any of the navigable childs "floor" and in the "navigation" window scroll down and click on "generate navigability"
